css/index.css

```diff
@@ -1,3 +1,352 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
 
+
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
```

Comment:
Thank you for not spreading this out one word per line.. Hate Whichever formatter does that